components/tidb_query_datatype/src/codec/row/v2/row_slice.rs

```diff
@@ -233,6 +233,9 @@ impl RowSlice<'_> {
             RowSlice::Big {
                 offsets, values, ..
             } => {
+                if non_null_col_num == 0 {
+                    return values.slice;
```

Comment:
`*values = LeBytes::new(&slice[..0])` before return, otherwise the `values` still point to the checksum part (although there might be no access to `values` since non_null_col_num is 0).

Reply:
It's refactored, PTAL again thx